src/App.tsx

```diff
@@ -1,7 +1,50 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  isPressedKey: boolean,
+  currentKey: string,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    isPressedKey: false,
+    currentKey: '',
+  };
```

Comment:
Here the same. Just `currentKey: '',` would be enough. It's already a falsy value.